mandatory/3-function-output.js

```diff
@@ -2,15 +2,18 @@
 function getRandomNumber() {
   return Math.random() * 10;
 }
+// this function  means return a random number between 0(inclusive), and 10(exclusive)
```

Comment:
Good job adding the comments 
Comments should'nt be added at the bottom of your code. that is bad practise